code/datums/emergency_calls/usm.dm

```diff
@@ -0,0 +1,61 @@
+/datum/emergency_call/usm
+	name = "USM Valor Marine"
+	pref_name = "EDF Marine"
+	weigh = 25
+	landmark_tag = "edfteam"
+
+/datum/emergency_call/usm/print_backstory(mob/living/carbon/human/H)
+	to_chat(H, "<B>You are USM Valor Marine EarthGov.</b>")
+	to_chat(H, "")
+	to_chat(H, "<B>Follow any orders directly from EarthGov Central Command.</b>")
+
+/datum/emergency_call/usm/create_member(datum/mind/M)
```

Comment:
And again, copypasting. This isn't how overriding should be done, this behaviour needs to be in base class and controlled by variables